test.env.example

```diff
@@ -1,4 +1,11 @@
-BIGQUERY_TEST_DATABASE=
-BIGQUERY_TEST_ALT_DATABASE=
-BIGQUERY_TEST_NO_ACCESS_DATABASE=
-BIGQUERY_TEST_SERVICE_ACCOUNT_JSON='{}'
+# Note: These values will come from your BigQuery account and GCP projects.
+
+# Test Environment field definitions
+BIGQUERY_TEST_DATABASE:
+# Name of the GCP project your dbt project is using.
+BIGQUERY_TEST_NO_ACCESS_DATABASE:
+# Name of a GCP project you don't have access to query.
+BIGQUERY_TEST_ALT_DATABASE:
+# Name of a second GCP project on your account.
+BIGQUERY_TEST_SERVICE_ACCOUNT_JSON:
+# Authentication method required to hookup to BigQuery via client library.
```

Comment:
sorry, one last request! can you put the description of the field above it?

so instead of
```
BIGQUERY_TEST_DATABASE:
# Name of the GCP project your dbt project is using.
```
it looks like
```
# Name of the GCP project your dbt project is using.
BIGQUERY_TEST_DATABASE:
```

Reply:
Done!